huaweicloud/services/acceptance/dbss/resource_huaweicloud_dbss_rds_database_test.go

```diff
@@ -0,0 +1,157 @@
+					resource.TestCheckResourceAttr(rName, "type", "MYSQL"),
+					resource.TestCheckResourceAttr(rName, "status", "ON"),
+					resource.TestCheckResourceAttrSet(rName, "db_id"),
+					resource.TestCheckResourceAttrSet(rName, "name"),
+					resource.TestCheckResourceAttrSet(rName, "version"),
+					resource.TestCheckResourceAttrSet(rName, "charset"),
+					resource.TestCheckResourceAttrSet(rName, "ip"),
+					resource.TestCheckResourceAttrSet(rName, "port"),
+					resource.TestCheckResourceAttrSet(rName, "os"),
+					resource.TestCheckResourceAttrSet(rName, "db_classification"),
+				),
+			},
+			{
+				Config: testAccAddRdsDatabase_update(name),
+				Check: resource.ComposeTestCheckFunc(
+					rc.CheckResourceExists(),
+					resource.TestCheckResourceAttr(rName, "status", "OFF"),
+				),
+			},
+			{
+				ResourceName:      rName,
+				ImportState:       true,
+				ImportStateVerify: true,
+				ImportStateVerifyIgnore: []string{
+					"lts_audit_switch",
+				},
+				ImportStateIdFunc: testAccAddRdsDatabaseImportState(rName),
+			},
+		},
+	})
+}
+
+// Before test, you need to set the default security group rule, enable `3306` port
+func testAccAddRdsDatabase_base(name string) string {
+	return fmt.Sprintf(`
+%[1]s
+
+data "huaweicloud_rds_flavors" "test" {
+  db_type       = "MySQL"
+  db_version    = "8.0"
+  instance_mode = "single"
+  group_type    = "dedicated"
+  vcpus         = 4
+}
+
+resource "huaweicloud_rds_instance" "test" {
+  name              = "%[2]s"
+  flavor            = data.huaweicloud_rds_flavors.test.flavors[0].name
+  security_group_id = data.huaweicloud_networking_secgroup.test.id
+  subnet_id         = data.huaweicloud_vpc_subnet.test.id
+  vpc_id            = data.huaweicloud_vpc.test.id
+  availability_zone = [data.huaweicloud_availability_zones.test.names[0]]
+
+  db {
+    type    = "MySQL"
+    version = "8.0"
+  }
+
+  volume {
+    type = "CLOUDSSD"
+    size = 100
+  }
+}
+`, testInstance_basic(name), name)
+}
+
+func testAccAddRdsDatabase_basic(name string) string {
+	return fmt.Sprintf(`
+%[1]s
+
+resource "huaweicloud_dbss_rds_database" "test" {
+  instance_id = huaweicloud_dbss_instance.test.instance_id
+  rds_id      = huaweicloud_rds_instance.test.id
+  type        = "MYSQL"
+  status      = "ON"
+}
+`, testAccAddRdsDatabase_base(name))
+}
+
+func testAccAddRdsDatabase_update(name string) string {
+	return fmt.Sprintf(`
+%[1]s
+
+resource "huaweicloud_dbss_rds_database" "test" {
+  instance_id = huaweicloud_dbss_instance.test.instance_id
+  rds_id      = huaweicloud_rds_instance.test.id
+  type        = "MYSQL"
+  status      = "OFF"
+}
+`, testAccAddRdsDatabase_base(name))
+}
+
+func testAccAddRdsDatabaseImportState(rName string) resource.ImportStateIdFunc {
+	return func(s *terraform.State) (string, error) {
+		var instanceId, rdsId string
+		rs, ok := s.RootModule().Resources[rName]
+		if !ok {
+			return "", fmt.Errorf("resource (%s) not found: %s", rName, rs)
```

Comment:
Variable `rs` is always nil in this line.
```suggestion
			return "", fmt.Errorf("resource (%s) not found", rName)
```